dockerfiles/ci/buster/php-5.5/Dockerfile

```diff
@@ -69,25 +94,40 @@ RUN set -eux; \
     for phpVer in $(ls ${PHP_INSTALL_DIR}); \
     do \
         # Set default INI settings
-        sed 's/;date.timezone =/date.timezone = UTC/' <${PHP_SRC_DIR}/php.ini-development >${PHP_INSTALL_DIR}/${phpVer}/etc/php.ini; \
+        sed 's/;date.timezone =/date.timezone = UTC/' <${PHP_SRC_DIR}/php.ini-development >${PHP_INSTALL_DIR}/${phpVer}/php.ini; \
         \
         echo "Install exts for PHP $phpVer..."; \
         switch-php ${phpVer}; \
+        pecl channel-update pecl.php.net; \
         iniDir=$(php -i | awk -F"=> " '/Scan this dir for additional .ini files/ {print $2}'); \
         \
         yes 'no' | pecl install memcached-2.2.0; echo "extension=memcached.so" >> ${iniDir}/memcached.ini; \
-        pecl install mongo; echo "extension=mongo.so" >> ${iniDir}/mongo.ini; \
+# Install mongo
+        export MONGO_VERSION="1.6.16"; \
+        export MONGO_SHA256="67886b696c428c9313539adc05f01bd87cabd57d2e861fef3c18d95375661fb2"; \
+        install-ext-from-source \
+            https://github.com/mongodb/mongo-php-driver-legacy/archive/refs/tags/${MONGO_VERSION}.tar.gz \
+            ${MONGO_SHA256} \
+            "--enable-mongo"; \
+        echo "extension=mongo.so" >> ${iniDir}/mongo.ini; \
+# Install xdebug
+        export XDEBUG_VERSION="2.5.5"; \
+        export XDEBUG_SHA256="72108bf2bc514ee7198e10466a0fedcac3df9bbc5bd26ce2ec2dafab990bf1a4"; \
+        install-ext-from-source \
+            https://xdebug.org/files/xdebug-${XDEBUG_VERSION}.tgz \
+            ${XDEBUG_SHA256} \
+            "--enable-xdebug"; \
         # Xdebug is disabled by default
-        pecl install xdebug-2.5.5; \
     done;
 
 RUN set -eux; \
 # Set the default PHP version
     switch-php debug;
 
 # Install Composer
-COPY --from=composer:2 /usr/bin/composer /usr/local/bin/composer
+COPY --from=composer:1 /usr/bin/composer /usr/local/bin/composer
 
 COPY welcome /etc/motd
 
-CMD ["php-fpm", "-F"]
+
+CMD [ "bash" ]
```

Comment:
It was. Since these are dev images I do not see why we should run php-fpm instead of bash as the default command, especially since I do not think php-fpm is configured to do anything interesting here.

But of course I have no strong opinions on this, so I am totally fine with reverting this change.

Reply:
It looks like this change was only made to the base image and the PHP 5.5 image, correct?

If we run `php-fpm` as the default command it makes it easier to use these containers with OOTB server containers like nginx. For example, we rely on this functionality for the [release-candidate checker](https://github.com/DataDog/dd-trace-php/blob/master/dockerfiles/release-candidates/docker-compose.yml#L35-L43) and I rely on it for debugging sometimes.

So if you don't have any strong options about it, do you mind if we keep the default command `php-fpm`? :)

Reply:
Absolutely no problem. Done here https://github.com/DataDog/dd-trace-php/commit/9bd79899e082299c4e95605f3ca4e7b399bda62b

Reply:
Also, I removed from the child containers that inherit it from the base.

Reply:
Thank you! Just one thought: `php-fpm` doesn't exist in the base container. Is that going to be an issue?

Reply:
not for docker (I mean if you do not use the base image directly). If you feel like this is confusing, I can add back to every image. Not a problem at all.